examples/keras_io/vision/image_classification_using_global_context_vision_transformer.py

```diff
@@ -0,0 +1,1248 @@
+"""
+Title: Image Classification using Global Context Vision Transformer
+Author: Md Awsafur Rahman
+Date created: 2023/10/30
+Last modified: 2023/10/30
+Description: Implementation and fine-tuning of Global Context Vision Transformer for image classification.
+"""
+"""
+# Setup
+"""
+
+"""shell
+pip install --upgrade keras_core
+pip install --upgrade keras_cv
+"""
+
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras_core as keras
+from keras_cv.layers import DropPath
+from keras_core import ops
+
+import jax
```

Comment:
Actually, I needed `jax/tf/torch` to build a model with dummy input. When I tried to avoid it using `model.build()`, I ran into a series of errors. I was able to bypass them using `ops.array()` for the dummy input.

Errors I faced:

1. When I used `mod.build()`, it didn't throw any errors. However, when I tried to load the weights, it indicated that the `head` and `norm` layers (layer index: -1 & -3) had not been built.

```shell
ValueError: Layer 'head' was never built and thus it doesn't have any variables. However, the weights file lists 2 variables for this layer.
In most cases, this error indicates that either:

1. The layer is owned by a parent layer that implements a `build()` method, but calling the parent's `build()` method did NOT create the state of the child layer 'head'. A `build()` method must create ALL state for the layer, including the state of any child layers.

2. You need to implement the `def build_from_config(self, config)` method on layer 'head', to specify how to rebuild it during loading. In this case, you might also want to implement the method that generates the build config at saving time, `def get_build_config(self)`. The method `build_from_config()` is meant to create the state of the layer (i.e. its variables) upon deserialization.
```

2. When I manually built them using,
```py
    def build(self, input_shape):
        super().build(input_shape)
        self.norm.build((None, 7, 7, 512))
        self.head.build((None, 512))
        self.built = True
```
the pre-trained checkpoints didn't work as they classified an `Egyptian cat` as an airplane or sometimes a bus. So, I compared the weights from 1) the model built with dummy input and 2) the model built with `model.build()`. Initially, I presumed that since `head` and `norm` had been built manually, they might be the culprits. However, it turned out to be the opposite. The mismatch was with the rest of the layers' weights. I double-checked and confirmed that the shape of the weights was exactly the same.

I also noticed that after a sanity check failure for pre-trained weights (with cat classification from an image), if I loaded the weights again and then performed another sanity check, it passed. This indicates that explicit input (dummy input) builds/initializes variables that `model.build()` somehow can't do.

To bypass these errors, I simply used the backend-agnostic `ops.array()` to build the model.